snapd.go

```diff
@@ -422,7 +457,6 @@ func action(ctx *cli.Context) {
 	if autodiscoverPath != "" {
 		log.Info("auto discover path is enabled")
 		paths := filepath.SplitList(autodiscoverPath)
-		c.SetAutodiscoverPaths(paths)
```

Comment:
I removed it because I noticed GetAutoDiscoverPaths() was part of the managesMetrics interface mgmt/rest/server.go. I didn't think it made sense to keep it in the interface when implementing RPC since it wasn't used. I decided to remove it from pluginControl because it didn't contain any special logic for handling autodiscover paths. It was only a getter/setter like below:
```golang
func (p *pluginControl) SetAutodiscoverPaths(paths []string) {
	p.autodiscoverPaths = paths
}

func (p *pluginControl) GetAutodiscoverPaths() []string {
	return p.autodiscoverPaths
}
```